lib/NewDataList/NewDataList.js

```diff
@@ -38,6 +38,10 @@ module.exports = kind({
 
 		e = event.type;
 
+		if((e === 'onSpotlightUp' && this.direction == 'horizontal') || (e === 'onSpotlightLeft' && this.direction == 'vertical')) {
```

Comment:
* Add a space between `if` and `(`
* Let's combine this condition with the one above for `isAccelerating()`
* We need to check for either direction (up or down with horizontal and left or right with vertical). You can still repro the bug by going to the *last* row, highlighting the first item, and holding down the right key.